pkg/pipeline/PipelineBuilder.go

```diff
@@ -1413,6 +1418,24 @@ func (impl PipelineBuilderImpl) PatchCiPipeline(request *bean.CiPatchRequest) (c
 	ciConfig.AppWorkflowId = request.AppWorkflowId
 	ciConfig.UserId = request.UserId
 	if request.CiPipeline != nil {
+		client, err := impl.K8sUtil.GetClientForInCluster()
+		if err != nil {
+			impl.logger.Errorw("exception while getting unique client id", "error", err)
+			return nil, err
+		}
+		cm, err := impl.K8sUtil.GetConfigMap(argo.DEVTRONCD_NAMESPACE, DashboardConfigMap, client)
+		if err != nil {
+			impl.logger.Errorw("error while getting dashboard-cm", "error", err)
+			return nil, err
+		}
+		datamap := cm.Data
+		forceScanConfig, err := strconv.ParseBool(datamap["FORCE_SECURITY_SCANNING"])
```

Comment:
add nil check for datamap and add constant for KEY

Reply:
done